base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java

```diff
@@ -154,8 +160,8 @@ public String getErrorMessage(WizardInfo wizardInfo) {
 
     public boolean send(String host, int port, String servlet, String rawData,
       WizardInfo wizardInfo) {
-        try {
-            Socket socket = new Socket(host, port);
+        try (Socket socket = new Socket(host, port);
```

Comment:
This is cool

Reply:
I am a fan of `try-with-multiple-resources` although I admit it sends me to Stack Overflow every time. I always forget that you separate the resources with a semicolon and not a comma, "Why won't you compile" etc :smile: